ror.yml

```diff
@@ -0,0 +1,147 @@
+# Recipe for Ruby on Rails collections
+---
+rh-ror50:
+  name: Ruby on Rails 5.0
+  requires: [rh-ruby23, rh-nodejs4, rh-mongodb32]
+  # install: Copr - Settings - Build options - Click "Edit" on target
+  #   Packages
+  install:
+    - scl-utils-build
+    - rh-ruby23-scldevel
+    - rh-ror50-build
+  packages:
+    - rh-ror50
+    # Packages required by RSpec, Cucumber
+    - rubygem-rspec
+    - rubygem-rspec-core:
+        replaced_macros:
+          need_bootstrap_set: 1
+    - rubygem-rspec-support:
+        replaced_macros:
```

Comment:
@junaruga, I'm just wondering, whether there is a semantic difference between `replaced_macros` and `macros` keys or whether it means the same. Does it mean the same or is there a difference?